### PR TITLE
[DOC] Add version example

### PR DESCRIPTION
**Pull request by bastians**
Added version example to general process of importing that it is more clear if you have to or wants to do the steps manually.

### CODE REVIEW

File: Documentation/AdministratorManual/InstallingExtension/Index.rst [hunk 1]
```diff
@@ -58,7 +58,7 @@ is stored as :file:`typo3temp/tx_sphinx/IMPORT-{date}.log`. The general process
 #. Fetch the version as a zip archive from https://github.com/sphinx-doc/sphinx/releases into
    directory :file:`typo3temp/`
 
-#. Unpack the zip archive into directory :file:`uploads/tx_sphinx/{version}/`
+#. Unpack the zip archive into directory :file:`uploads/tx_sphinx/{version}/` (name of the folder should be `v.1.8.1`, e.g.)
```
Comment (xperseguers) [inline Documentation/AdministratorManual/InstallingExtension/Index.rst]: name is not `v.<something>` but `v<something>`:

```
..... (name of the folder should be e.g., `v1.8.1`)
```

**Review by xperseguers — CHANGES_REQUESTED** [review]
Please change commit message to something like

```
[DOC] Add version example
```

to respect the contribution guidelines.

**Comment by xperseguers** [comment]
BTW, please switch to branch `TYPO3_v7`, master branch is starting not to be compatible anymore with TYPO3 7.6.

Would love if you can test the version on TYPO3 v7 so that I could possibly release one very last version on TER.

**Comment by bastians** [comment]
Thanks for the review. Fixed the version schema and changed the title of the PR.

**Comment by xperseguers** [comment]
OK, squashed the commits, reworked it a bit and merge to both `master` and `TYPO3_v7`branches.

**Comment by bastians** [comment]
> BTW, please switch to branch `TYPO3_v7`, master branch is starting not to be compatible anymore with TYPO3 7.6.
> 
> Would love if you can test the version on TYPO3 v7 so that I could possibly release one very last version on TER.

I am on it, but I still get a message regarding unzip but still I believe it's not available on my server or not configured correctly. Will continue.

**Comment by bastians** [comment]
Ok, downloading and unzipping works now, but now I am facing some problems with building. Never ending story. I guess again some server issues and nothing related to the extension.

> [CMD] '/usr/bin/python' -V 2>&1
> Python 2.7.9
> [CMD] cd '/is/htdocs/wp10621052_KBL8Q09Y2Z/www/cms/typo3/prod/uploads/tx_sphinx/v1.8.1' && '/usr/bin/python' setup.py clean 2>&1 && '/usr/bin/python' setup.py build 2>&1
> Traceback (most recent call last):
>   File "/usr/lib/python2.7/site.py", line 563, in <module>
>     main()
>   File "/usr/lib/python2.7/site.py", line 545, in main
>     known_paths = addusersitepackages(known_paths)
>   File "/usr/lib/python2.7/site.py", line 272, in addusersitepackages
>     user_site = getusersitepackages()
>   File "/usr/lib/python2.7/site.py", line 247, in getusersitepackages
>     user_base = getuserbase() # this will also set USER_BASE
>   File "/usr/lib/python2.7/site.py", line 237, in getuserbase
>     USER_BASE = get_config_var('userbase')
>   File "/usr/lib/python2.7/sysconfig.py", line 582, in get_config_var
>     return get_config_vars().get(name)
>   File "/usr/lib/python2.7/sysconfig.py", line 533, in get_config_vars
>     _CONFIG_VARS['userbase'] = _getuserbase()
>   File "/usr/lib/python2.7/sysconfig.py", line 210, in _getuserbase
>     return env_base if env_base else joinuser("~", ".local")
>   File "/usr/lib/python2.7/sysconfig.py", line 196, in joinuser
>     return os.path.expanduser(os.path.join(*args))
>   File "/usr/lib/python2.7/posixpath.py", line 269, in expanduser
>     userhome = pwd.getpwuid(os.getuid()).pw_dir
> KeyError: 'getpwuid(): uid not found: 10621052'

**Comment by xperseguers** [comment]
Ticket is closed, open a new one if willing to keep up debugging “publicly”. Problem is possibly related to using docker or some other container (just googled a bit the end of your stack trace).

**Pull request closed without merging**